### PR TITLE
Client vpn work

### PR DESCRIPTION
**Pull request by dkeppel626**
Added client VPN discovery

### CODE REVIEW

File: aws-inventory/lambda/inventory-client-vpn.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+from common import *
+
+import logging
+logger = logging.getLogger()
+logger.setLevel(getattr(logging, os.getenv('LOG_LEVEL', default='INFO')))
+logging.getLogger('botocore').setLevel(logging.WARNING)
+logging.getLogger('boto3').setLevel(logging.WARNING)
+logging.getLogger('urllib3').setLevel(logging.WARNING)
+
+RESOURCE_PATH = "ec2/vpn"
```
Comment (jchrisfarris) [inline aws-inventory/lambda/inventory-client-vpn.py]: Lets call this clientvpn to distinguish between site-to-site VPNs.

File: aws-inventory/lambda/inventory-client-vpn.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    response = ec2_client.describe_client_vpn_endpoints()
+    
+    if response['ClientVpnEndpoints']:
+        
+        for cvpn in response['ClientVpnEndpoints']:
+            
+            resource_item = {}
+            resource_item['awsAccountId']                   = target_account.account_id
+            resource_item['awsAccountName']                 = target_account.account_name
+            resource_item['resourceType']                   = "AWS::EC2::ClientVpnEndpoints"
```
Comment (jchrisfarris) [inline aws-inventory/lambda/inventory-client-vpn.py]: Please make this ClientVpnEndpoint (singular) to match the CF resource type:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-clientvpnendpoint.html

File: aws-inventory/lambda/inventory-client-vpn.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            resource_item['errors']                         = {}
+        
+            if 'Tags' in cvpn:
+                resource_item['tags']                       = parse_tags(cvpn['Tags'])
+
+           # Get any active VPN connections to the endpoint and add as part of the supplmentry configuration.
+            connections = discover_client_vpn_connections(ec2_client, cvpn['ClientVpnEndpointId'])
+            resource_item['supplementaryConfiguration']['Connections'] = connections
+    
+            # Obtain other network configuration associated with the VPN endpoint and add as part of the supplmentry configuration.
```
Comment (jchrisfarris) [inline aws-inventory/lambda/inventory-client-vpn.py]: supplmentry -> supplementary

**Review by jchrisfarris — CHANGES_REQUESTED** [review]

File: aws-inventory/lambda/inventory-client-vpn.py (hunk 1 above)
Comment (dkeppel626) [reply aws-inventory/lambda/inventory-client-vpn.py]: updated

File: aws-inventory/lambda/inventory-client-vpn.py (hunk 3 above)
Comment (dkeppel626) [reply aws-inventory/lambda/inventory-client-vpn.py]: updated

File: aws-inventory/lambda/inventory-client-vpn.py (hunk 2 above)
Comment (dkeppel626) [reply aws-inventory/lambda/inventory-client-vpn.py]: updated

**Comment by dkeppel626** [comment]
One additional update, changed the vpn index to clientvpn index in the post_deploy.sh script to match changes done in the inventory client vpn script.

**Review by jchrisfarris — APPROVED** [review]

**Pull request merged**